src/scripts/choices.js

```diff
@@ -569,7 +569,9 @@ class Choices {
               console.error(err);
             }
           })
-          .then(() => this._handleLoadingState(false))
+          .then(() =>
+            requestAnimationFrame(() => this._handleLoadingState(false)),
+          )
```

Comment:
I'm not a fan of mixing promises and callbacks, as my head stops understand flow. Can we just return PromiseLike here if we must wait for animation frame?
```js
  .then(() => ({ then: next => requestAnimationFrame(next) }))
  .then(() => this._handleLoadingState(false));
```

Reply:
@tinovyatkin For me no difference, it's not mine codebase, the only thing I need is to make it working properly. :)